mbed-hal-efm32/PeripheralNames.h

```diff
@@ -60,6 +60,9 @@ typedef enum {
     PWM_CH0 = 0,
     PWM_CH1 = 1,
     PWM_CH2 = 2,
+#ifdef TIMER_ROUTEPEN_CC0PEN
```

Comment:
Needs to be changed to CC3PEN to be relevant, otherwise this is just canonical to P2